sync/class.jetpack-sync-queue.php

```diff
@@ -339,7 +339,7 @@ private function get_next_data_row_option_name() {
 			$this->row_iterator += 1;
 		}
 
-		return 'jpsq_' . $this->id . '-' . $timestamp . '-' . getmypid() . '-' . $this->row_iterator;
+		return 'jpsq_' . $this->id . '-' . $timestamp . '-' . random_int( 1, 1000000 ) . '-' . $this->row_iterator;
```

Comment:
I would either remove the row iterator from the key or move random_int above and use the same random_int for every row. Doing it that way would also be computationally cheaper.